tests/NewCypress/cypress/pages/trusts/academiesInTrustPage.ts

```diff
@@ -0,0 +1,85 @@
+class AcademiesInTrustPage {
+
+    elements = {
+        DetailsPage: {
+            table: () => cy.get('.govuk-main-wrapper > .dfe-width-container > .govuk-grid-row > .govuk-grid-column-three-quarters'),
+            authTypes: () => cy.get('[data-testid="type-of-establishment"]'),
```

Comment:
this isn't selecting anything to do with auth (or local authorities?)

Reply:
These were supposed to be school types and not auth types - I must have had a brain moment when naming this and the element

Now updated to reflect school type not auth type